cmd/flags.go

```diff
@@ -98,6 +98,32 @@ var GrafanaFlags = []cli.Flag{
 	},
 }
 
+// DockerFlags are used when producing docker images.
+var DockerFlags = []cli.Flag{
+	&cli.StringFlag{
+		Name:  "registry",
+		Usage: "Prefix the image name with the registry provided",
+	},
+	&cli.BoolFlag{
+		Name:  "save",
+		Usage: "Use 'docker save' to save the image as a tar archive. If this flag is set then the --destination flag will be used",
```

Comment:
IMO, if a destination is set, then the tarball should be written there. No need for a `--save` flag.

Reply:
I agree